services/brig/src/Brig/User/Search/Index/Types.hs

```diff
@@ -109,6 +114,7 @@ mkIndexUser u v =
   IndexUser
     { _iuUserId = u,
       _iuVersion = v,
+      _iuAccountStatus = Nothing,
```

Comment:
Minor style nit: `_iuAccountStatus` is the last line of the record in all other places.